docs/modules/operation/pages/admin/webui/mib.adoc

```diff
@@ -0,0 +1,39 @@
+
+= SNMP MIB Compiler
+
+Management Information Base (MIB) files are text files written in xref:https://en.wikipedia.org/wiki/ASN.1[Abstract Syntax Notation].
+They describe Simple Network Management Protocol (SNMP) elements for specific devices.
+By default, {page-component-title} includes an extensive list of MIB definitions from a number of vendors.
+You will only need to import MIB files for unformatted events as part of configuring SNMP traps and collection.
+
+You can parse SNMP MIB definitions and create {page-component-title} event definitions from any `TRAP-TYPE` and/or `NOTIFICATION-TYPE` macros they contain through the UI.
+You can also generate data collection definitions from `OBJECT-TYPE` symbols defined in a MIB file.
+
+Two command line utilities, `mib2events` and `mib2opennms`, are also available to help make event definitions from MIB definitions.
```

Comment:
```suggestion
Two command line utilities, `mib2events` and `mib2opennms`, are also available to help create event definitions from MIB definitions.
```

Reply:
mib2events was archived a long time ago:  https://github.com/OpenNMS-Archives/mib2events

I don't think we need to keep any reference to it in the docs; it's dead.